Cargo.toml

```diff
@@ -49,3 +49,7 @@ inherits = "test"
 opt-level = 2
 [profile.dev.package.adler]
 opt-level = 1
+
+[patch.crates-io]
+tauri = { git = "https://github.com/davide-baldo/tauri.git", branch="add-url-scheme" }
+tauri-build = { git = "https://github.com/davide-baldo/tauri.git", branch="add-url-scheme" }
```

Comment:
Could we please fork this into the build-trust GitHub org if possible? And mark a tracking issue that we can comment above this, so we'll know in the future when it's safe to remove again?

Reply:
makes perfect sense, done: 
https://github.com/build-trust/ockam/issues/5667
https://github.com/build-trust/tauri